index.js

```diff
@@ -96,14 +96,35 @@ class ResembleHelper extends Helper {
     });
   }
 
+  /**
+   * Help function for date and time format
+   * @param number
+   * @returns number
+   */
+  _pad(number) {
+    if (number < 10) {
+      return `0${number}`;
+    }
+    return number;
+  }
+
+  /**
+   * Get actual date and time in format MMMM-MM-MMTHH-MM-SS
+   * @returns <void>
+   */
+  _getTimestamp() {
+    let now = new Date();
+    return `${now.getFullYear()}-${this._pad(now.getMonth() + 1)}-${this._pad(now.getDate())}T${this._pad(now.getHours())}:${this._pad(now.getMinutes())}:${this._pad(now.getSeconds())}`;
```

Comment:
The ISO format must be in UTC timezone. Otherwise you must specify the timezone offset by adding `Z+02:00` at the end of the string.
See the reference here: https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC
This is in the fact wrong currently. Because you use the getHours() function which is returning the hours in our timezone. And then you paste in the ISO format which is saying to user the time is in the UTC zero. But it's not. 
So we've got two options. Just use standart `toISOstring()` and leave it in UTC or you must add into the output also the specification for the timezone.